src/lib/notation.ts

```diff
@@ -27,4 +27,29 @@ export function parseNotation(algString: string): MoveSeq {
 
 function isValidMove(move: string): move is Move {
   return isRotation(move as Move) || isLayerMove(move as Move)
+}
+
+function isValidMoveVC(move: string): boolean {
+  if("weiosdfghjkl".search(move)) return true
+  return false
+}
+export function isValidVC(vc: string): boolean {
+  return vc === "" || vc.trim().split("").every(isValidMoveVC)
+}
+export function parseVC(vc: string): Move {
+  const moveTable: {[name: string]: string} = {
+    "w": "B",
+    "e": "L'",
+    "i": "R",
+    "o": "B'",
+    "s": "D",
+    "d": "L",
+    "f": "U'",
+    "g": "F'",
+    "h": "F",
+    "j": "U",
+    "k": "R'",
+    "l": "D'"
+  }
+  return (moveTable[vc]??"") as Move;
```

Comment:
This type assertion is dangerous because empty string isn't a `Move`. I suggest returning null to indicate the letter did not correspond to a move

Reply:
oops forgot that null exists